### ISSUE TITLE
Verification

### ISSUE THREAD

**Issue by le000043** [Jannoshh/simple-sam#4]
Have you conducted experiments to verify that your implementation could reproduce similar results to the original implementation?
Thanks

**Comment by Jannoshh** [comment]
I've only conducted small scale experiments, mostly on MNIST. The results seem to suggest that this SAM implementation works, you can look at the results in the examples directory.
The results definitely verify: 
- less overfitting, can train for more epochs without overfitting
- higher accuracy even when training with half the epochs

The implementation is also very similar to https://github.com/davda54/sam which seems to work great and has been verified in many experiments.
